src/components/TodosFooter.tsx

```diff
@@ -0,0 +1,64 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+
+export enum FILTERS {
+  All = 'all',
+  Active = 'active',
+  Completed = 'completed',
+}
+
+type Props = {
+  filterBy: FILTERS,
+  activeTodosLength: number,
+  completedTodosLength: number,
+  onFilterChange: (filterBy: FILTERS) => void
+  onDeleteCompleted: () => void
+};
+
+export const TodosFooter: FC<Props> = ({
+  filterBy,
+  activeTodosLength,
+  completedTodosLength,
+  onFilterChange,
+  onDeleteCompleted,
+}) => {
+  const handleOnDelete = () => {
+    if (completedTodosLength) {
+      onDeleteCompleted();
+    }
+  };
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${activeTodosLength} Items left`}
```

Comment:
if there are more then 2 items - text should be `${activeTodosLength} items left` if only one item left text should be `${activeTodosLength} item left`